react/components/Combobox.css

```diff
@@ -0,0 +1,15 @@
+:root {
+  --reach-combobox: 1;
+}
+
+.option:not([aria-selected="true"]).option:not(:hover) {
+  background-color: transparent;
+}
+
+.option:active {
+  background-color: #d2defc
+}
```

Comment:
@davicosta99, while addressing the following issue

> When a suggestion is selected (on:active) it should blink in the on-action color token (#D2DEFC) for 50ms before closing.

I found out that this color does not exist on [VTEX Tachyons spec](https://github.com/vtex/vtex-tachyons/blob/master/tachyons.css), so I’ve added it directly here.

Also, how do I know if it takes 50ms during the blink? Is that the standard behavior?

Reply:
Same goes for `#c4c4c4` on Place Icon and `#2e2e2e` on Warning Icon, but I used the closest "c-muted-x" for them:
- `#c4c4c4` -> `#cacbcc` (`c-muted-3`)
- `#2e2e2e` -> `#727273` (`c-muted-1`)

Reply:
Currently we have these tokens, that basically reuse the hover color.
.active-bg-action-secondary:active { background-color: #dbe9fd; }
.active-b--action-secondary:active { border-color: #dbe9fd; }

Can you change them to #D2DEFC? This change was made in Figma but never implemented.

Regarding the 50ms, I thought this was something that needed to be defined. Right now is perfect already.

Sorry for the mistake on the grays, my bad. Thanks for using the right tokens 😉